spec/requests/citizens/consents_spec.rb

```diff
@@ -1,11 +1,53 @@
 require 'rails_helper'
 
-RSpec.describe 'citizen consents request test', type: :request do
-  describe 'GET /citizens/consent' do
-    before { get citizens_consent_path }
+RSpec.describe Applicants::OpenBankingConsentForm, type: :request do
```

Comment:
This spec is describing the output from a controller action. It is not a direct test of the form. So:
```
RSpec.describe Citizens::ConsentsController, type: :request do
```